hook/scriptwrapper.go

```diff
@@ -180,6 +162,36 @@ func NewScriptWrapper(opts ...scriptWrapperOpt) (*ScriptWrapper, error) {
 	return wrap, nil
 }
 
+func (wrap *ScriptWrapper) GenerateCmdDotExeScript(pathToHook string) string {
```

Comment:
I believe we call these batch scripts elsewhere

Reply:
Ooh good call!